src/hooks/useInflation.ts

```diff
@@ -76,7 +76,11 @@ export function useInflation(): UseInflation {
       }
 
       const balancesRepository = container.get<IBalancesRepository>(Symbols.BalancesRepository);
-      const initialTotalIssuance = await balancesRepository.getTotalIssuance(period1StartBlock - 1);
+      const initialTotalIssuance =
+        (await balancesRepository.getTotalIssuance(period1StartBlock - 1)) -
+        (networkNameSubstrate.value.toLowerCase() === 'astar'
+          ? BigInt('350000000000000000000000000')
+          : BigInt(0)); // Quick fox for token burning event. TODO make a proper solution
```

Comment:
What about not showing if it's Shiden or Shibuya?

Reply:
In that case we will deduce 0n which is fine because no burning happened on Shiden, Shibuya

> What about not showing if it's Shiden or Shibuya?

In that case we will deduce 0n which is fine because no burning happened on Shiden, Shibuya

Reply:
> What about not showing if it's Shiden or Shibuya?

In that case we will deduce 0n which is fine because no burning happened on Shiden, Shibuya